eth/tracers/native/prestate.go

```diff
@@ -116,8 +120,12 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to commo
 
 // CaptureEnd is called after the call finishes to finalize the tracing.
 func (t *prestateTracer) CaptureEnd(output []byte, gasUsed uint64, _ time.Duration, err error) {
-	if t.create && !t.config.DiffMode {
-		// Exclude created contract.
+	if t.config.DiffMode {
+		return
+	}
+
+	if t.create && !t.pre[t.to].isExists() {
```

Comment:
Shouldn't you do a similar `if s := t.pre[a]; s != nil {` ?

Reply:
sorry ：）

Reply:
Seems we don't need to care about the `to_address` nullable, in `diffMode`, the created address may be cleared if nothing is modified. But in prestate-only mode, we don't need to care about that.


But for consistency, let's check it.